src/charts/stackedArea/stackedAreaChart.js

```diff
@@ -45,4 +51,32 @@ stackedArea.create = (el, data, configuration = {}) => {
     return chart;
 };
 
+stackedArea.update = (el, data, configuration = {}) => {
+    let container = select(el);
+    let configurationProperties = Object.keys(configuration);
+
+    validateContainer(container);
+    validateConfiguration(chart, configurationProperties);
+
+    // Sets up the passed configuration
+    configurationProperties.forEach((key) => {
```

Comment:
this might be a nice named helper method

Reply:
The configuration setting or the validations?